ui/desktop/tests/acceptance/authentication-test.js

```diff
@@ -15,10 +22,17 @@ import {
   invalidateSession,
 } from 'ember-simple-auth/test-support';
 import WindowMockIPC from '../helpers/window-mock-ipc';
+import setupStubs from 'api/test-support/handlers/cache-daemon-search';
 
 module('Acceptance | authentication', function (hooks) {
   setupApplicationTest(hooks);
   setupMirage(hooks);
+  setupStubs(hooks);
+
+  const SIGNOUT_BTN_SELECTOR = '[data-test-nav-signout-btn]';
+  const CLOSE_SESSIONS_MODAL_SELECTOR = '[data-test-close-sessions-modal]';
+  const CONFIRM_BUTTON = '.hds-modal__footer .hds-button--color-primary';
+  const CANCEL_BUTTON = '.hds-modal__footer .hds-button--color-secondary';
```

Comment:
For naming conventions here can we stick to `BTN` instead of `BUTTON` like in line 32. And I think the new convention is to leave out the word `SELECTOR` similar to line 35.